README.md

```diff
@@ -178,7 +212,7 @@ You have to prepare a directory, with files in format `0xabc..93f.json` that con
 Then you have to add `--override-bytecodes-dir=XX` flag to point at that directory. See the `example_override` dir for more details.
 
 ```bash
-cargo run -- --override-bytecodes-dir=example_override --show-storage-logs all fork mainnet
+cargo run -- --override-bytecodes-dir=example_override --show-storage-logs all fork --fork-url mainnet
```

Comment:
Any reason this refers to `cargo run` instead of `anvil-zksync`?

Reply:
No will update